docs/src/main/paradox/getting-started.md

```diff
@@ -0,0 +1,107 @@
+}
+
+Add the following imports to your project:
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideSourceProviderImports }
+
+Create the @apidoc[SourceProvider].
+The @ref:[Event Sourced Source Provider](eventsourced.md) is using [Akka Persistence](https://doc.akka.io/docs/akka/current/typed/persistence.html) internally (specifically the [eventsByTag](https://doc.akka.io/docs/akka/current/persistence-query.html#eventsbytag-and-currenteventsbytag) API).
+To initialize the Source Provider we need to set parameters to choose the Akka Persistence plugin (Cassandra) to use as well as the name of the tag used for events we're interested in from the journal.
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideSourceProviderSetup }
+
+## Part 2: Build a simple Projection handler
+
+While building a projection there are several non-functional requirements to consider.
+_What technology to project into? What message delivery semantics are acceptable for the system? Is it compatible with the chosen Source Provider to enable exactly-once message delivery? Does runtime state need to be maintained in the projection while it's running?_
+It's up to the user to choose the right answers to these questions, but you must research if the answers to these questions compatible with each other.
+
+For the purpose of this guide we will persist our Projection to Cassandra with at-least-once semantics and state will not be required in the projection handler itself.
+To proceed we must add the Cassandra Projection implementation to our project:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-projection-cassandra_$scala.binary.version$
+  version=$project.version$
+}
+
+Add the following imports to your project:
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideProjectionImports }
+
+It's the user's responsibility to implement the means to project into the target system, the Projection will only manage the persistence of the offset (though it is possible to enlist your projection into transactions when using projection implementations that support exactly-once like the @ref:[JDBC](jdbc.md)).
+Write the code necessary to implement your projection.
+For this example we are going filter `ShoppingCartEvents.CheckedOut` events from the Source Provider and write them to a Cassandra table that can be used to see the current daily count, as well as historical counts for the number of checked out shopping carts for each day of the year.
+Implement a Cassandra repository that will manage a Cassandra table named `daily_checkouts`.
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideProjectionRepo }
+
+Now it's time to write the Projection handler itself.
+There are many kinds of handlers we can choose from.
```

Comment:
I would remove this sentence. I think that, unless we link to a list of "kinds of handlers" it's not providing valuable information.

Reply:
I agree, a getting started guide can show one way without going in to all alternatives